commandapi-platforms/commandapi-velocity/commandapi-velocity-core/src/main/java/dev/jorel/commandapi/CommandAPIVelocityConfig.java

```diff
@@ -1,20 +1,25 @@
 package dev.jorel.commandapi;
 
+import com.velocitypowered.api.plugin.Plugin;
 import com.velocitypowered.api.proxy.ProxyServer;
 
 /**
  * A class that contains information needed to configure the CommandAPI on Velocity-based servers.
  */
 public class CommandAPIVelocityConfig extends CommandAPIConfig<CommandAPIVelocityConfig> {
+	Object plugin;
 	ProxyServer server;
 
 	/**
 	 * Creates a new CommandAPIVelocityConfig object. Variables in this
 	 * constructor are required to load the CommandAPI on Velocity properly.
 	 *
+	 * @param plugin The plugin object (annotated by {@link Plugin}) loading the CommandAPI.
+	 *               This is used when registering events.
 	 * @param server The {@link ProxyServer} that the CommandAPI is running on.
 	 */
-	public CommandAPIVelocityConfig(ProxyServer server) {
+	public CommandAPIVelocityConfig(Object plugin, ProxyServer server) {
```

Comment:
In #526, the plugin is the second parameter.
Should this be updated to reflect that change or do we want to change that here again?
```diff
- public CommandAPIVelocityConfig(Object plugin, ProxyServer server) {
+ public CommandAPIVelocityConfig(ProxyServer server, Object plugin) {
```
This would reflect my change from #526, however, I do not know if you've already updated this branch to contain the Velocity plugin namespaces but I guess you didn't.
I just wanted to mention this so we can talk about that if necessary.

Reply:
Yeah, this branch is currently conflicting with the changes from #526. I don't think it matters which way around the parameters go, so when I rebase this branch I'll just stick with what is currently on `dev/dev`.